lib/libc/gen/Symbol.map

```diff
@@ -522,6 +522,7 @@ FBSDprivate_1.0 {
 	_sleep;
 	_rtld_allocate_tls;
 	_rtld_free_tls;
+	__emutls_get_address;
 #if defined(i386)
```

Comment:
Wait this gets preprocessed? Maybe we should have done `#if defined(__aarch64__) && defined(__CHERI_PURE_CAPABILITY__)`. Oh well, I don't really care that much, emutls's days are hopefully numbered.

Reply:
I hope it's days are numbered as well, and am just glad it's all one commit we can git revert when the time comes.

Reply:
Hopefully we'll have removed it by the time someone thinks to implement errors for symbols that don't exist. I'm pretty sure I found typos in a sweep a few years ago...